lib/app/pages/create_folder_page/widgets/country_button.dart

```diff
@@ -30,25 +35,24 @@ class _CountryButtonState extends State<CountryButton> {
         setState(() {
           _selectedCountryCode = country.countryCode;
         });
+
+        widget.onSelect(_selectedCountryCode);
+        setState(() {});
```

Comment:
you already have a set state above, you don't need to rebuild the widget two times